### ISSUE TITLE
Problem with gpx start and end  values in json

### ISSUE THREAD

**Issue by SapozhnikUA** [mapillary/mapillary_tools#478]
### Basic information
* Release version: `0.8.0`
* System: ``Linux` 
* Capture Device:  `DVR`

### Steps to reproduce behavior
 1. I have folder with ~ 100 different mp4 and gpx files
 2. I created bash which run mapillary_tools string for each file:
mapillary_tools video_process "$video" "$IMG_PATH" --geotag_source "gpx" --geotag_source_path "$gpx_path" --interpolate_directions --video_sample_interval 0.5 --overwrite_EXIF_gps_tag --interpolation_use_gpx_start_time --skip_process_errors
 3. MT creates a lot folders with different names and images for each video files and one json-file in main folder
 4. If we open json we can see that gpx start and end  values the same for all image-files with error
                "gpx_start_time": "2021_11_20_13_27_47_000",
                "gpx_end_time": "2021_11_20_13_28_46_000"
`        "error": {
            "type": "MapillaryOutsideGPXTrackError",
            "message": "Unable to interpolate: the image timestamp is beyond the GPX trace end point by 0.5 seconds",
            "vars": {
                "image_time": "2021_11_20_13_28_46_500",
                "gpx_start_time": "2021_11_20_13_27_47_000",
                "gpx_end_time": "2021_11_20_13_28_46_000"
            }
`
I uploaded json file and several video and gpx files there: [example](https://drive.google.com/drive/folders/1FoQOWUHpV4Pk-RCnu47Rz0iiPYMqwN7S)

**Comment by ptpt** [comment]
It's likely to be fixed in https://github.com/mapillary/mapillary_tools/issues/473

**Comment by SapozhnikUA** [comment]
> It's likely to be fixed in #473
> 
> Could you update to the latest and try again
> 
> ```
> python3 -m pip install --force-reinstall --upgrade git+https://github.com/mapillary/mapillary_tools
> ```
Hello, 
Thank you for reply.

I updated it again but it doesn't help
I see the same data in json and at the end:

```
2021-11-23 11:17:18,856 - INFO   - {
    "total_images": 10200,
    "processed_images": 110,
    "failed_images": 10081,
    "duplicated_images": 9
}
2021-11-23 11:17:18,856 - WARNING - Skipping 10081 failed images
2021-11-23 11:17:18,857 - INFO   - Check /mnt/drive_d/mapillary/img/mapillary_image_description.json for details

```
